src/service/run-api-service.js

```diff
@@ -12,6 +12,7 @@
  *
  * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects).
  * * `project`: Epicenter project id.
+ * * `id`: (Optional) The run id of an existing run. If there is a specific run you'd like to work with, you can include it.
```

Comment:
yes, it does appear above the config section.
to clarify though -- what would it mean to create a run service with a filter that returns several runs? that e.g. if you called rs.do('myOper') (<-- note no options arg) you'd be calling it on everything in the filter?